scripts/install_custom_dc.sh

```diff
@@ -14,7 +14,14 @@
 # limitations under the License.
 set -e
 
-CUSTOM_DC_RELEASE_TAG=custom-dc-v0.1.0
+CUSTOM_DC_RELEASE_TAG=custom-dc-v0.2.0
```

Comment:
Maybe update the tag after it's created?

Reply:
currently custom-dc-v0.1.0 is used for the previous version of the script, if we update it after, we are changing the script for custom-dc-v0.1.0.